server/src/dtos/auth.dto.ts

```diff
@@ -78,6 +78,28 @@ export class ChangePasswordDto {
   newPassword!: string;
 }
 
+export class createPincodeDto {
+  @IsString()
+  @IsNotEmpty()
+  @Length(6, 6)
```

Comment:
What's the reason you want to limit it to 6 chars? Do you want to build one of those shiny UI elements that's not actually a plain text input?